src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
+$baza: 40px;
+
 body {
   margin: 0;
+  padding: 20px;
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 730px;
+  
+  & :first-child {
+    margin-left: 622px;
+  }
+
+  @for $index from 1 through 31 {
+    & :nth-child(#{$index})::before {
+      position: absolute;
+      content: "#{$index}";
+      width: 20px ;
+      height: 20px;
+      left: $baza;
+      top: $baza;
+      font-family: Arial, sans-serif;
```

Comment:
it's better not to add `font-family` here. instead you can add this style bo `body`